elasticjob-lite/elasticjob-lite-lifecycle/src/main/java/org/apache/shardingsphere/elasticjob/lite/lifecycle/internal/settings/JobConfigurationAPIImpl.java

```diff
@@ -31,22 +31,26 @@
  */
 @RequiredArgsConstructor
 public final class JobConfigurationAPIImpl implements JobConfigurationAPI {
-    
+
```

Comment:
BTW, keep the indents please.

Reply:
fixed，sorry but there is a litte strange why  4 space indent in every empty line